user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityAndPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
+There isn't currently any formal review process for add-ons performed by a human.

```

Comment:
I do not consider this sentence to be accurate in a few critical respects:

* Use of "formal", implies that there may be "informal" reviews going on, that we just don't talk about.
* And what are "reviews", anyway? There may be user generated reviews of some kind, but there are no security reviews. No code reviews. No functionality reviews. No documentation reviews. The whole store is based on Trust. It doesn't even enforce the licensing to be done in a certain way, just trusts the testimony of the submitter in the submission form that it matches the license selected.
* Use of "yet" indicates future plans to do the opposite of whatever is "not yet happening", and NV Access has made no such commitment. This is not a printed document, where we need to hint at the future because things might change before we can publish an update. The document will evolve in concert with reality, so there is no need to predict and hint.

Of course, once you strip away the vagaries from this sentence, it becomes just a different wording of what is already clearly stated in the "Installing Add-ons" section of this same chapter.
In fact, I encourage you to read that section, as much of what you're saying here, is already said there.

```suggestion
Add-ons are not reviewed by NV Access staff before publication to the Add-on Store.
```